docs/orchestrating-elastic-stack-applications/stack-config-policy.asciidoc

```diff
@@ -0,0 +1,236 @@
+:page_id: stack-config-policy
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Elastic Stack configuration policies
+
+NOTE: This requires a valid Enterprise license or Enterprise trial license. Check <<{p}-licensing,the license documentation>> for more details about managing licenses.
+
+Starting from ECK `2.6.0` and Elasticsearch `8.6.0`, Elastic Stack configuration policies let you configure in a declarative way, for multiple clusters, the following settings:
```

Comment:
```suggestion
Starting from ECK `2.6.0` and Elasticsearch `8.6.0`, Elastic Stack configuration policies allow you to configure the following settings:
```

declarative is implied imo, and multiple clusters is mentioned after the settings list again